nx.json

```diff
@@ -25,15 +26,10 @@
       },
       "projectChangelogs": true
     },
-    "groups": {
```

Comment:
A single group is no longer required, we can use the projects shorthand and not need the nested structure (and not need to give it a name)